frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepTable.tsx

```diff
@@ -0,0 +1,152 @@
+            const color = getStepColor(step, !!filters.breakdown)
+            return (
+                <SeriesToggleWrapper
+                    id={step.order}
+                    toggleVisibility={() => step.breakdown && setVisibility(step.breakdown, !isVisible)}
+                    style={{ display: 'flex', alignItems: 'center' }}
+                >
+                    {isBreakdownChild ? (
+                        <div style={{ maxWidth: 270, wordBreak: 'break-word' }}>
+                            {formatBreakdownLabel(step.breakdown, cohorts)}
+                        </div>
+                    ) : (
+                        <div style={{ flexGrow: 1 }}>
+                            <InsightLabel
+                                seriesColor={color}
+                                fallbackName={step.name}
+                                hasMultipleSeries={steps.length > 1}
+                                breakdownValue={step.breakdown}
+                                hideBreakdown
+                                hideIcon
+                            />
+                        </div>
+                    )}
+                </SeriesToggleWrapper>
+            )
+        },
+        fixed: 'left',
+        width: 300,
+    })
+
+    columns.push({
+        title: 'Completed',
+        render: function RenderCompleted({}, step: FlattenedFunnelStep): JSX.Element {
+            return (
+                <ValueInspectorButton onClick={() => openPersonsModal(step, step.count + 1, step.breakdown_value)}>
+                    {step.count}
+                </ValueInspectorButton>
+            )
+        },
+        width: 30,
+    })
+
+    columns.push({
+        title: 'Conversion',
+        render: function RenderConversion({}, step: FlattenedFunnelStep): JSX.Element | null {
+            return step.order === 0 ? null : <span>{step.conversionRates.total}%</span>
+        },
+        width: 30,
+    })
+
+    columns.push({
+        title: 'Dropped off',
+        render: function RenderDropoff({}, step: FlattenedFunnelStep): JSX.Element | null {
+            return step.order === 0 ? null : (
+                <ValueInspectorButton
+                    onClick={() =>
+                        openPersonsModal(step, step.count + 1, step.breakdown_value)
+                    } /* TODO: does this modal support dropped off users? */
+                >
+                    {step.droppedOffFromPrevious}
+                </ValueInspectorButton>
+            )
+        },
+        width: 30,
+    })
+
+    columns.push({
+        title: 'From previous',
+        render: function RenderDropoffFromPrevious({}, step: FlattenedFunnelStep): JSX.Element | null {
+            return step.order === 0 ? null : <span>{humanizeNumber(100 - step.conversionRates.fromPrevious, 2)}%</span>
+        },
+        width: 30,
+    })
+
+    columns.push({
+        title: 'Mean time',
```

Comment:
We use "Average" instead of mean everywhere except when providing additional details to better explain this number